src/main/java/com/redhat/devtools/intellij/knative/actions/func/RunAction.java

```diff
@@ -52,6 +53,7 @@ public void actionPerformed(AnActionEvent anActionEvent, TreePath path, Object s
 
         ExecHelper.submit(() -> {
             try {
+                telemetry.property(PROP_CALLER_ACTION, "run func");
```

Comment:
I think I was not clear enough. I want to know which action triggered another action without user request (ex run trigger a build) so in telemetry for build action I should have a property which have run as value

Reply:
yeah this should do the job. When the run action is clicked, it adds a property to the telemetry object which is then passed to the build action. When it gets executed and send the event, the property is attached and it's possible to see which was the action which triggered it, no?

Reply:
If it wasn't triggered by anything, the property doesn't exist

Reply:
Ah you want to send a completely different event "run trigger build". Got it